src/lib/utils.js

```diff
@@ -436,3 +436,9 @@ function parseAroundLatLngFromString(value) {
     lng: parseFloat(pattern[2]),
   };
 }
+
+export function getPropertyByPath(object, path) {
```

Comment:
I would unit test that :)

Reply:
https://github.com/algolia/vue-instantsearch/blob/feat/connectors/src/util/__tests__/object.test.js